src/reporters/base.js

```diff
@@ -77,15 +77,16 @@ export default class BaseReporter {
         // This happens because the next test can't be completed until the
         // previous one frees all browser connections.
         // Therefore, tests always get completed sequentially.
-        var reportItem = this.reportQueue.shift();
-        var durationMs = new Date() - reportItem.startTime;
+        var reportItem     = this.reportQueue.shift();
+        var durationMs     = new Date() - reportItem.startTime;
+        var screenshotPath = reportItem.screenshotCreated ? reportItem.screenshotPath : null;
```

Comment:
What is `reportItem.screenshotCreated`? Maybe it should be named `reportItem.hasScreenshots`?